.rubocop.yml

```diff
@@ -5,7 +5,7 @@ AllCops:
     - vendor/**/*
     - Gemfile.lock
     - Rakefile
-  TargetRubyVersion: 2.4
+  TargetRubyVersion: 2.7
```

Comment:
Not cheked with rubocop as rubocop is not in the Gemfile (shall we remove this file?)